rpm/libphonenumber.spec

```diff
@@ -70,6 +72,7 @@ Contains files needed to development with %{name}.
        -DUSE_LITE_METADATA=ON \
        -DUSE_RE2=OFF \
        -DBUILD_STATIC_LIB=OFF \
+       -DCMAKE_BUILD_TYPE=Release \
```

Comment:
This feels like a bug on its own that it needs to be enabled explicitly. Also worth to note it enables `-O3`.

Reply:
Changed to `RelWithDebInfo`, which does not enables `-O3`. The leading note still applies - shouldn't that be the default?

Reply:
Sure it feels like a bug, but then again CMake documentation (https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#default-and-custom-configurations) says "Users should always explicitly specify the build type" so this is in line with that.